### ISSUE TITLE
[Color Scheme - Camphell]: Name is not defined updated if user selects any other color.

### ISSUE THREAD

**Issue by Priyanka-Chauhan123** [microsoft/terminal#15983]
### Windows Terminal version

1.19.2481.0

### Windows build number

23529.1000

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 23529.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Repro Steps: **

1.Open Windows Terminal.
2.Open Settings page using 'Ctr+,'
3. Navigate to the 'Color Scheme' and Select 'Camphell'.
4. Now navigate to any color and changed it according to your will.
5. Navigate over the color using Narrator and observe the issue.

**User Experience: **
Users who have cerebral palsy and in-active limbs and depend on Voice Access for their accessibility will not be able to access the options with its visible name if the accessible name defined is incorrect.

MAS Reference:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Attachment :
[Name is not defined updated if user selects any other color..zip](https://github.com/microsoft/terminal/files/12644638/Name.is.not.defined.updated.if.user.selects.any.other.color.zip)

### Expected Behavior

Name should get updated if user selects any other color.

### Actual Behavior

Name is not defined updated if user selects any other color.

**Comment by zadjii-msft** [comment]
This and #15985 are the same thing - just kinda a misunderstanding of how Terminals work. You're totally free to set "Black"(and the rest of the palette) to whatever color _value_ you want. That however doesn't change the meaning of those color palette entries to CLI applications. 

This seems like by design to me honestly.

**Comment by DHowett** [comment]
This is correct. @Priyanka-Chauhan123, "Black", "Red", "White" and the other 13 colors in the Terminal settings are terms of art. The standard language for terminal emulators defines 16 colors that have specific names. The user is allowed to redefine how those names appear, but the names may never change.

**Comment by DHowett** [comment]
@zadjii-msft for 15985, I wonder if it would be possible to indicate "Foreground (Ochre)" or "Background (Vermillion)"?

**Comment by Priyanka-Chauhan123** [comment]
@carlos-zamora By when it is expected to get fixed, as it is impacting product compliance

**Comment by DHowett** [comment]
@Priyanka-Chauhan123 please see questions above directed towards you. These are not intended to be the names of the user's selected color, they are intended to be the name of the standard "color slot" the user is changing.

**Comment by Priyanka-Chauhan123** [comment]
@DHowett Does it meant that user will not be able to change the color and it will not be modified. If this is the case, then it should not change the color visibly as well.

**Comment by DHowett** [comment]
@Priyanka-Chauhan123 It is different. In this case, "red" is a symbolic name used as part of an API contract. It's like syntax highlighting, with tokens for "error" or "filename" or "string".

We allow the user to change to meaning of the symbolic constant ("red"), just like Visual Studio lets you change the color of "String", without changing its name. Changing its name would break the API contract.

Does that help?
